build/crd/crunchy/generated/postgres-operator.crunchydata.com_postgresclusters.yaml

```diff
@@ -8352,6 +8352,8 @@ spec:
                 properties:
                   pgAudit:
                     type: boolean
+                  pgRepack:
```

Comment:
@pooknull  Do we need to create helm chart PR for this new option?

Reply:
and I think we need to add test for all builtin extensions.

Reply:
https://perconadev.atlassian.net/browse/K8SPG-755

Reply:
https://github.com/percona/percona-helm-charts/pull/515